hazelcast/src/main/java/com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -666,6 +659,11 @@ private void finalizeOrRollbackMigration(PartitionRuntimeState partitionState) {
         }
 
         updateAllPartitions(partitionTable);
+        partitionStateManager.setVersion(partitionState.getVersion());
+        if (!partitionStateManager.setInitialized()) {
```

Comment:
Is there a chance that the partition state manager is not initialized? Can we not just call `node.getNodeExtension().onPartitionStateChange();`?